doc/whats_new/v0.21.rst

```diff
@@ -458,6 +458,13 @@ Support for Python 3.4 and below has been officially dropped.
   useless or resulting in a wrong approximation of the cumulative distribution
   function estimator. :issue:`13333` by :user:`Albert Thomas <albertcthomas>`.
 
+- |API| Deprecate default `copy=False` value of
```

Comment:
this seems strange, maybe copy the deprecation message here.
``preprocessing.data.quantile_transform`` -> ``preprocessing.quantile_transform``